package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "preact-enzyme-adapter",
+  "name": "enzyme-adapter-preact",
   "version": "0.1.0",
   "description": "A Preact adapater for the Enzyme testing utility",
-  "main": "lib/index.js",
+  "main": "src/index.js",
```

Comment:
Actually this is incorrect; the file is build output and doesn’t exist in Github but does exist on prepublish. Please revert this.

Reply:
Altho i see there’s not actually a build process; perhaps the proper fix is to revert this along with adding a build process?

Reply:
Yes there's no build process afaict, and the version I got from `npm install` has no /lib or /lib/index.js

Reply:
Example:
```$ npm i enzyme-adapter-preact
+ enzyme-adapter-preact@0.1.0
added 1 package in 1.516s
$ more node_modules/enzyme-adapter-preact/
.babelrc      __tests__/    package.json  src/          yarn.lock ```

Reply:
Added separate PR for this: #11